test.rs

```diff
@@ -0,0 +1,41 @@
+mod common;
+
+// Include tests directly in this file if desired
+#[cfg(test)]
+mod tests {
+    // use rust_vmmap::*;
+
+    use crate::common;
+
+    #[test]
+    fn tests() {
+        common::setup();
+        //TODO: implement integration test
+
+        let mut vmmap = Vmmap::new();
+        
+        // Test adding a valid entry (should pass)
```

Comment:
Split each test into its own test, only club when it's a part of the same test pipeline (see vmmap.rs)

Reply:
Got it. Thanks!